### ISSUE TITLE
Add Clone content to other post types

### ISSUE THREAD

**Issue by jlyon** [proudcity/wp-proudcity#1443]
See https://github.com/proudcity/wp-proudcity/issues/1438

Duplicate Post currently enabled on 
Page
News

It _does_ appear to copy metafields. Q: Do we want it on other post types? If so, which?
Meetings
Events
Documents
...

**Comment by lukefretwell** [comment]
@jlyon yeah, I think these are great, as well as jobs if we don't already have it there. Imagine there's a lot of dupe info.

**Comment by kevindherman** [comment]
@jlyon the ability to clone jobs just came up. Is adding the duplication ability a quick item?

**Comment by curtismchale** [comment]
@kevindherman combing through issues while you're out, yes this is easy. The settings are here: https://beta.proudcity.com/wp-admin/options-general.php?page=duplicatepost

Let me know what post types you want the ability to clone and I can get a script together to deploy the settings platform wide.
